### DISCUSSION TITLE
UI/UX layout

### DISCUSSION THREAD

**Discussion by lrmodesgh** [opengisch/QField#2217] [Ideas]
[Qfield needs better UI/UX that ensures reliability a pillar of FOSS ecosystem]

As a FOSS/FOSH supporter, i feel that QGIS, OSGeo, OSM, HOT have done so much good, amid the internal community level frictions all these years.
Given the demand for spatial and temporal information among the big corporates, resisting exploitative business model is very hard. And i respect opengisch for that to a great extent, and designers/developers community like me will also support in all the way we can including donation from our earned salary, documenting, designing frameworks, suggestions for improvements, popularizing it among the communities involved in participative mapping, bridging state bureaucracy to use the app, etc...

Thus from a design and utility point of view, i think, a major revision is needed in Qfield layout.
Based upon my experience with FOSS apps, i feel that MapTiler android app has one of the simple yet powerful design features and layour. Next to that is Trekarta. OSMAnd+ is a complete thing, but must not be compared to Qfield.

I wont suggest core level changes, but any laout level features, components, assets, colors, etc... will be very useful.
If any help is needed on UI layout design i would like to contribute my time through designing in inkscape svg, which developers can take it for real implementation.

Please consider this as enhancement suggestion

**Comment by nirvn** [comment]
@lrmodesgh , this is pretty close to opening an issue saying "Your UI sucks!". I can't see much "constructivity" in that. 

Few points you should take into consideration here:

1/ The QField project is trying to be _the best version of QField it can be_; it is _not_ interested in emulating the looks / behavior of other apps merely for the sake of looking like the other apps. While nobody will argue the UI is perfect as is, there are plenty of deliberate, informed decisions that were made over the years to shape the way QField is looking now and that needs to be respected by at the very least offering arguments when suggesting changes. 

2/ Some of what you're describing is quite subjective; to pick one: colors :-) we happen to love QField's lime green, it's been attached to the app's identity. It's subjective, some will like, some won't, but that's something that really can't be argued here. 

That being said, if you want to highlight _particular_ UI elements which you think could feature a better icon or behave better, feel free to open _specific_ issues on those, with a clear rational to explain yourself. You've mentioned being capable of designing inkscape, feel very welcome to provide improved icons for the ones you think could be improved.

**Reply by lrmodesgh** [reply]
@nirvn sorry for being so blunt...

i didnt intend to criticize on the labour that went into deciding the UI as well as UX. Also i didnt want to compare the other apps as is. I just communicated what my peers who dont have design/development ideas in the field. Also i dont want to emulate other apps. But i felt that there are few things that can be learned. To be included or not, that the collaborative process will decide. I am happy about that.

However, as u guided me to use my design skill for contributing some of the icons, i will get into that soon.

Also i will next time will ensure to explain my rational in a more specific and reasonable way, instead of ranting on a broad scope.

Thanks @nirvn for getting back immediately.

**Reply by m-kuhn** [reply]
Thanks for the feedback @lrmodesgh 

Excited to see creative initiatives!

**Comment by ZsanettMed** [comment]
Hello @lrmodesgh ,

We've set up the [QField Idea Platform](https://ideas.qfield.org/app-feature-requests) for improvement suggestions. Feel free to share your thoughts there or send us an email at [sales@qfield.org](mailto:sales@qfield.org). Looking forward to your inputs!

Best, 
Zsanett
